modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,277 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
 
 In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.8.
 
+[id="pipelines-new-features-1-8_{context}"]
+=== Pipelines
+
 :FeatureName: Running {pipelines-title} on ARM hardware
-[id="running-pipelines-on-arm-hardware-1-8_{context}"]
-=== {FeatureName}
+* {FeatureName} With this update, you can run {pipelines-title} GA 1.8 and later on an {product-title} cluster that is running on ARM hardware. This includes support for `ClusterTask` resources and the `tkn` CLI tool.
```

Comment:
There's an issue rendering this text in the preview build. Not sure what's going on:

![image](https://user-images.githubusercontent.com/86380613/185698204-d200a6a9-ee45-4e05-a3bf-ea74e46ec06a.png)

Reply:
Updated. Thanks.